### DISCUSSION TITLE
Who can provide me with a simple example code.

### DISCUSSION THREAD

**Discussion by clearlove75** [gempy-project/gempy#938] [Q&A]
Could someone simply write a piece of code to generate a geological interface that contains random geological events. I had a lot of problems with my code, and I tried for a long time but it didn't work. Always report errors, such as inability to import or parameter passing problems.
My version of gempy is 2024.2.0.2 and python is 3.12.4.

**Comment by javoha** [comment]
Hi,
your question is very general. The code you send me has a lot of issues unrelated to your problem. I think you should spend some time to review the gempy tutorials and understand the modeling approach and the syntax.
Regarding your idea: Just passing random points to gempy will not work due to the nature of the underlying universal cokriging approach. It is possible to shift inputa data points in certain ranges, but ivery broadly speaking it requires a sensible starting model.
Cheers
Jan
